src/uct/ib/rc/base/rc_iface.c

```diff
@@ -81,6 +76,15 @@ static ucs_stats_class_t uct_rc_iface_stats_class = {
 };
 #endif
 
+
+static ucs_mpool_ops_t uct_rc_fc_pending_mpool_ops = {
+    .chunk_alloc   = ucs_mpool_hugetlb_malloc,
```

Comment:
no need for hugetlb - FC is not fast path, we would just consume too much memory here